client-app/shared/cart/components/cart-add-info.vue

```diff
@@ -74,6 +92,8 @@ const props = defineProps({
 
 const variant = computed(() => (props.lineItem.quantity === 0 ? "warn" : "success"));
 const title = computed(() =>
-  props.lineItem.quantity === 0 ? "1 Product removed from cart" : "1 Product added to cart"
+  props.lineItem.quantity === 0
+    ? t("shared.cart.cart_add_info_popup.title_1")
```

Comment:
Uninformative title keys: please, use "title_removed" and "title_added" instead.